test/Microsoft.AspNetCore.Mvc.FunctionalTests/HtmlGenerationTest.cs

```diff
@@ -98,7 +98,7 @@ public class HtmlGenerationTest :
 #else
                 // Mono issue - https://github.com/aspnet/External/issues/19
```

Comment:
Same a few more times in this class, `HtmlHelperOptionsTest` and `HtmlHelperCheckboxTest`.